chia/full_node/weight_proof.py

```diff
@@ -124,6 +124,9 @@ async def _create_proof_of_weight(self, tip: bytes32) -> WeightProof | None:
             return None
 
         summary_heights = self.blockchain.get_ses_heights()
+        if len(summary_heights) <= 1:
+            log.error("failed not enough sub epochs")
```

Comment:
get_seed_for_proof  expects at least two sub epochs, it will throw without this.
we cannot get a sees if we dont have 2 finished sub epochs at least

we cannot change how the seed is chosen since it is verified by the verifier so we can change this without being incompatible